crates/hir-def/src/macro_expansion_tests/mbe.rs

```diff
@@ -35,9 +35,9 @@ macro_rules! f {
     };
 }
 
-struct#0:1@58..64#1# MyTraitMap2#0:2@31..42#0# {#0:1@72..73#1#
-    map#0:1@86..89#1#:#0:1@89..90#1# #0:1@89..90#1#::#0:1@91..93#1#std#0:1@93..96#1#::#0:1@96..98#1#collections#0:1@98..109#1#::#0:1@109..111#1#HashSet#0:1@111..118#1#<#0:1@118..119#1#(#0:1@119..120#1#)#0:1@120..121#1#>#0:1@121..122#1#,#0:1@122..123#1#
-}#0:1@132..133#1#
+struct#0:1@58..64#4# MyTraitMap2#0:2@31..42#2# {#0:1@72..73#4#
```

Comment:
We really ought to do something about these tests, in new Salsa they change every time someone create a new interned type and that is awful.

Reply:
Yes, we talked about that on zulip iirc, for new salsa a bunch of tests need to change the rendering of what they expect. For this kind of stuff we likely need to have the test allocate a map and assign stable IDs that way